### PR TITLE
[relay]: adds new relay log types

### PR DESCRIPTION
**Pull request by maraisr**
Adds in new network log types as per https://github.com/facebook/relay/commit/95606c5ed

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## relay-runtime ([<kbd>unpkg</kbd>](https://unpkg.com/browse/relay-runtime@latest/))
was missing the following properties:
1. ReplaySubject
1. getNode
1. reportMissingRequiredFields
1. isFragment
1. isInlineDataFragment

<details>
<summary>as well as these 13 other properties...</summary>
<p>isRequest, MutationTypes, RangeOperations, MutationHandlers, VIEWER_ID, VIEWER_TYPE, PreloadableQueryRegistry, createPayloadFor3DField, RelayError, DEFAULT_HANDLE_KEY, isScalarAndEqual, recycleNodesInto, stableCopy</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e982182a91a666e8068a8001486331acdda0a879
</p>

File: types/relay-runtime/lib/store/RelayStoreTypes.d.ts [hunk 1]
```diff
@@ -457,6 +457,35 @@ type LogEvent =
           name: 'execute.unsubscribe';
           transactionID: number;
       }>
+    | Readonly<{
+        name: 'network.info',
+        transactionID: number,
+        info: any,
```
Comment (alloy) [inline types/relay-runtime/lib/store/RelayStoreTypes.d.ts:463]: The TS `unknown` type semantically maps best to [Flow’s `mixed`](https://flow.org/en/docs/types/mixed/)

```suggestion
        info: unknown,
```

**Review by alloy — CHANGES_REQUESTED** [review]
One small request, otherwise looks good 👍

**Comment by alloy** [comment]
👍

**Pull request merged**